app/workers/users/record_field_test_event_worker.rb

```diff
@@ -9,21 +9,32 @@ def perform(user_id, experiment, goal)
       user = User.find(user_id)
 
       if goal == "user_views_article_four_days_in_week"
-        determine_pageview_goal(user, experiment)
+        determine_weekly_pageview_goal(user, experiment)
+      elsif goal == "user_views_article_four_of_past_7_hours"
+        determine_past_seven_hours_pageview_goal(user, experiment)
       else
         field_test_converted(experiment, participant: user, goal: goal)
       end
     end
 
     private
 
-    def determine_pageview_goal(user, experiment)
+    def determine_weekly_pageview_goal(user, experiment)
       past_week_page_view_counts = user.page_views.where("created_at > ?", 7.days.ago).
         group("DATE(created_at)").count.values
       past_week_page_view_counts.delete(0)
       return unless past_week_page_view_counts.size > 3
 
       field_test_converted(experiment, participant: user, goal: "user_views_article_four_days_in_week")
     end
+
+    def determine_past_seven_hours_pageview_goal(user, experiment)
+      past_seven_hours_counts = user.page_views.where("created_at > ?", 7.hours.ago).
```

Comment:
Something I thought about looking at this, my DEV usage is usually split throughout the day. I like to do a morning check and then an after-work check which means I am likely getting in 4 posts but not within 7 hours. I feel like that might be common for people that choose not to check it during the work day. What do you think about increasing this timeframe?

Reply:
Cool, that works for me. The 4/7 choice is fairly arbitrary and if anything just lined up with the days poetically.

I'll change it to 4/24 which is a fine timeframe and probably much easier for us to understand as observers of the test.